### ISSUE TITLE
metapac incorrectly triggers sudo when Flatpak Systemwide is set to "false" (0.2.5)

### ISSUE THREAD

**Issue by tommodev** [ripytide/pacdef#63]
Hello!

For Version: `metapac 0.2.5`

`config.toml` has the following set:

> flatpak_systemwide = false

However when executing `metapac clean`, metapac prompts for sudo and the following error returns:

```bash
[sudo] password for XXXXXX:
error: Refusing to operate under sudo with --user. Omit sudo to operate on the user installation, or use a root shell to operate on the root user's installation.
Error:
   0: command failed: "sudo flatpak uninstall --user <my flat packages>"

Location:
   src/cmd.rs:82

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

Assuming the behaviour should be to _not_ execute flatpak under sudo when Systemwide is set to false.

On testing this with "metapac sync", it looks like it respects the systemwide flag (i.e., does not prompt for sudo on install)

Thank you for the work reviving this app!

**Comment by ripytide** [comment]
Apologies, this was fixed in #57 but we haven't made a new release since as I've been working on some other big changes I wanted to do in the next release #62, I probably should have just done a small bug release right away and then released #62 in another release. Hopefully, now the #62 PR is ready for review a new `v0.2.6` release will be out soon.

**Comment by ripytide** [comment]
Actually, I don't think #62 is ready so I'll make a patch release in the meantime for #57.

**Comment by ripytide** [comment]
`v0.2.6` has now been released which fixes this :tada:

**Comment by tommodev** [comment]
> Apologies, this was fixed in #57 but we haven't made a new release since as I've been working on some other big changes I wanted to do in the next release #62, I probably should have just done a small bug release right away and then released #62 in another release. Hopefully, now the #62 PR is ready for review a new `v0.2.6` release will be out soon.

No need for apologies, app adds a tonne of value as-is. Thank you for getting the fix out and happy holidays!
